assets/training/model_management/src/azureml/model/mgmt/config.py

```diff
@@ -53,3 +53,36 @@ class LoggerConfig:
     VERBOSITY_LEVEL = "DEBUG"
     OFFLINE_RUN_MESSAGE = "OFFLINE_RUN"
     ASSET_NOT_FOUND = "AssetID missing in run details"
+
+
+class LlamaHFModels(_CustomEnum):
+    """Llama HF Models."""
```

Comment:
nit: For this use case think dictionary make more sense over an ENUM?

Reply:
added dictionary as well